frontend/src/scenes/plugins/edit/interface-jobs/PluginJobConfiguration.tsx

```diff
@@ -40,15 +40,17 @@ export function PluginJobConfiguration(props: InterfaceJobsProps): JSX.Element {
         <>
             <span className="ml-1" onClick={() => playButtonOnClick(jobHasEmptyPayload)}>
                 <Tooltip title={configureOrRunJobTooltip}>
-                    {jobHasEmptyPayload ? (
-                        <IconPlayCircle
-                            className={runJobAvailable ? 'Plugin__RunJobButton' : 'Plugin__RunJobButton--disabled'}
-                        />
-                    ) : (
-                        <IconSettings
-                            className={runJobAvailable ? 'Plugin__RunJobButton' : 'Plugin__RunJobButton--disabled'}
-                        />
-                    )}
+                    <span>
```

Comment:
if we have to wrap children in a span, should the toolbar do that so we don't need to remember when we use it?

Reply:
Tldr is that we shouldn't need to wrap every icon in a span. I've made changes to the `@posthog/icons` repo to support forward refs. Anywhere we use an icon within a tooltip shouldn't need a span if it's imported from `@posthog/icons`, all other icons inside a tooltip I've manually found / replaced in this PR. More changes coming in https://github.com/PostHog/posthog/pull/20378 that remove our reliance on old icons so this won't need to be a consideration for others going forward (might even merge that first so these spans can go too)